packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -81,7 +81,9 @@ export default class CallDiagnosticLatencies extends WebexPlugin {
       key === 'client.media.rx.start' ||
       key === 'client.media.tx.start' ||
       key === 'internal.client.meetinginfo.request' ||
-      key === 'internal.client.meetinginfo.response'
+      key === 'internal.client.meetinginfo.response' ||
+      key === 'internal.register.device.request' ||
+      key === 'internal.register.device.response'
```

Comment:
does this actually happen more than once ?

Reply:
I don't think it should, we can discuss